tests/unit_tests/test_geoip_service.py

```diff
@@ -0,0 +1,53 @@
+from aiohttp import web
+import gzip
+import os
+import random
+import string
+from unittest.mock import Mock
+
+import server.config
+from server.geoip_service import GeoIpService
+
+
+async def test_check_update(geoip_service):
+    path = '/tmp/fake_db.mmdb'
+    with open(path, 'a'):
+        os.utime(path, (0, 0))
+    server.config.GEO_IP_DATABASE_MAX_AGE_DAYS = 32
+    geoip_service.file_path = '/tmp/fake_db.mmdb'
+    geoip_service.load_db = Mock()
+
+    await geoip_service.check_update_geoip_db()
+
+    geoip_service.download_geoip_db.assert_called_once_with()
```

Comment:
assert_called_once ?

Reply:
Doesn't exist in python3.5. Need 3.6 for that :P

https://docs.python.org/3.5/library/unittest.mock.html#unittest.mock.Mock.assert_called_once_with